packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -857,9 +858,10 @@ private boolean isScrollPerfLoggingEnabled() {
   public void draw(Canvas canvas) {
     if (mEndFillColor != Color.TRANSPARENT) {
```

Comment:
This doesn't seem right, since we are now comparing int color to packed color long.

Reply:
I believe in this case they both evaluate to 0L but I'll update to make it explicit.